zad1/main.rb

```diff
@@ -1,13 +1,19 @@
 def sort_letters(string)
 
+  string.chars.sort.join()
```

Comment:
Jak metoda nie przyjmuje żadnego argumentu to w Ruby nie trzeba jej kończyć pustymi nawiasami ;)